projects/FW_102/src/main.c

```diff
@@ -0,0 +1,54 @@
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define A_COUNTER_MS 500
+#define B_COUNTER_MS 1000
```

Comment:
You only need the 500ms macro here since you removed the second loop, and the `A_` prefix should be removed